projects/kafka.apache.org/package.yml

```diff
@@ -0,0 +1,73 @@
+# vendored
+distributable:
+  # url: https://downloads.apache.org/kafka/{{version}}/kafka-{{version}}-src.tgz
+  url: https://downloads.apache.org/kafka/{{version}}/kafka_2.13-{{version}}.tgz # This only matters if you are using Scala and you want a version built for the same Scala version you use. Otherwise any version should work (2.13 is recommended).
+  strip-components: 1
+
+warnings:
+  - vendored
+
+provides:
+  - bin/connect-distributed.sh
+  - bin/connect-mirror-maker.sh
+  - bin/connect-plugin-path.sh
+  - bin/connect-standalone.sh
+  - bin/kafka-acls.sh
+  - bin/kafka-broker-api-versions.sh
+  - bin/kafka-client-metrics.sh
+  - bin/kafka-cluster.sh
+  - bin/kafka-configs.sh
+  - bin/kafka-console-consumer.sh
+  - bin/kafka-console-producer.sh
+  - bin/kafka-consumer-groups.sh
+  - bin/kafka-consumer-perf-test.sh
+  - bin/kafka-delegation-tokens.sh
+  - bin/kafka-delete-records.sh
+  - bin/kafka-dump-log.sh
+  - bin/kafka-e2e-latency.sh
+  - bin/kafka-features.sh
+  - bin/kafka-get-offsets.sh
+  - bin/kafka-jmx.sh
+  - bin/kafka-leader-election.sh
+  - bin/kafka-log-dirs.sh
+  - bin/kafka-metadata-quorum.sh
+  - bin/kafka-metadata-shell.sh
+  - bin/kafka-producer-perf-test.sh
+  - bin/kafka-reassign-partitions.sh
+  - bin/kafka-replica-verification.sh
+  - bin/kafka-run-class.sh
+  - bin/kafka-server-start.sh
+  - bin/kafka-server-stop.sh
+  - bin/kafka-storage.sh
+  - bin/kafka-streams-application-reset.sh
+  - bin/kafka-topics.sh
+  - bin/kafka-transactions.sh
+  - bin/kafka-verifiable-consumer.sh
+  - bin/kafka-verifiable-producer.sh
+  - bin/trogdor.sh
+  - bin/zookeeper-security-migration.sh
+  - bin/zookeeper-server-start.sh
+  - bin/zookeeper-server-stop.sh
+  - bin/zookeeper-shell.sh
+
+versions:
+  github: apache/kafka/tags
+
+dependencies:
+  openjdk.org: '*'
+
+build:
+  dependencies:
+    rsync.samba.org: '*'
+  script:
+    - rm -rf bin/windows
+    - sed -i 's|log.dirs=/tmp/kafka-logs|log.dirs={{prefix}}/kafka-logs|' config/server.properties
+    - sed -i 's|log.dirs=/tmp/kraft-combined-logs|log.dirs={{prefix}}/kraft-combined-logs|' config/server.properties
+    - sed -i 's|log.dirs=/tmp/kraft-controller-logs|log.dirs={{prefix}}/kraft-controller-logs|' config/controller.properties || true
+    - sed -i 's|log.dirs=/tmp/kraft-broker-logs|log.dirs={{prefix}}/kraft-broker-logs|' config/broker.properties || true
+    - sed -i 's|dataDir=/tmp/zookeeper|log.dirs={{prefix}}/zookeper|' config/zookeeper.properties
```

Comment:
this probably needs some work to be relocatable. `{{prefix}}` on the build machine won't exist on install machines. if this can be controlled by runtime.env vars (unlikely) we should do that, otherwise, we'll need to replace the scripts with a shim that fixes paths before launching. see https://github.com/pkgxdev/brewkit/blob/main/share/brewkit/python-venv-stub.py for the most complicated version.

Reply:
Yeah, there's no way to know the installed prefix, since it depends on where the user installs pkgx. Ideally, programs are runnable directly without pkgx once installed (not always true, but often true). Hence using a shim, so you can correct at runtime. Other tricks abound, such as using $(dirname $(dirname $0)) in shell scripts.

Reply:
actually, looking at what you're changing, it's probably fine to just remove the sed commands. using /tmp is probably optimal.

Reply:
I see, makes sense. Will have a look later

Reply:
we'll try it as /tmp for now, unless that breaks anything.